platform/process/tasks/ABProcessTaskServiceInsertRecord.js

```diff
@@ -85,28 +85,40 @@ module.exports = class InsertRecord extends InsertRecordTaskCore {
          tasks.push(
             Promise.resolve()
                .then(() => pullTask())
-               .then((val) =>
```

Comment:
Go back to using the `this._req.retry()` here.

However it needs to be:  `return this._req.retry(() => this.object.model().create(val));` for this to work correctly.